lib/theme-options/options-front-page.php

```diff
@@ -51,522 +54,638 @@ function nm_register_front_page_options_metabox() {
   $prefix = 'nm_';
 
   $banner_options = array(
-    false => 'None',
-    'partials/support-section' => 'Support section',
-    'partials/specials/banners/support-video' => 'Support Video',
-    'partials/specials/banners/podcast-if-i-speak' => 'Podcast: If I Speak',
-    'partials/specials/banners/focus-pro-rev-soccer' => 'Focus: Pro Rev Soccer',
-    'partials/specials/banners/podcast-foreign-agent' => 'Podcast: Foreign Agent',
-    'partials/specials/banners/focus-doing-it-right-sex-on-the-left' => 'Focus: Doing It Right: Sex On The Left',
-    'partials/specials/banners/focus-breaking-britain' => 'Focus: Breaking Britain',
-    'partials/specials/banners/focus-disability-its-political' => 'Focus: Disability: It’s Political',
-    'partials/specials/banners/podcast-planet-b' => 'Podcast: Planet B',
+      false                                              => 'None',
+      'partials/support-section'                         => 'Support section',
+      'partials/specials/banners/support-video'          => 'Support Video',
+      'partials/specials/banners/podcast-if-i-speak'     => 'Podcast: If I Speak',
+      'partials/specials/banners/focus-pro-rev-soccer'   => 'Focus: Pro Rev Soccer',
+      'partials/specials/banners/podcast-foreign-agent'  => 'Podcast: Foreign Agent',
+      'partials/specials/banners/focus-doing-it-right-sex-on-the-left' => 'Focus: Doing It Right: Sex On The Left',
+      'partials/specials/banners/focus-breaking-britain' => 'Focus: Breaking Britain',
+      'partials/specials/banners/focus-disability-its-political' => 'Focus: Disability: It’s Political',
+      'partials/specials/banners/podcast-planet-b'       => 'Podcast: Planet B',
+      'partials/specials/banners/podcast-committed'      => 'Podcast: Committed',
```

Comment:
Tend to add these in reverse chrono order but leaving the support stuff at the top. So better to put this above If I Speak